### DISCUSSION TITLE
Option to highlight on yank

### DISCUSSION THREAD

**Discussion by godalming123** [helix-editor/helix#10205] [Ideas]
In neovim you can make text get highlighted when you yank this help to make yanking text more clear, I could not find the option in the docs.

**Comment by gabydd** [comment]
As helix is select then act all the text that is getting yanked will already be highlighted, I don't see any reason to highlight it again

**Reply by chrisbohens** [reply]
The behavior of the vim yy can more or less be replicated in Helix. Try out yy with and then without yank highlighting and you will know why OP was asking for it.

**Comment by godalming123** [comment]
It makes things more clear especially when I try to use neovim muscle memory
